src/playbin.js

```diff
@@ -1,22 +1,23 @@
 /* exported Playbin */
 
-const {Gdk, GLib, GObject, Gtk, Gst} = imports.gi;
+const {GLib, GObject, Gtk, Gst} = imports.gi;
 
 var Playbin = GObject.registerClass({
     CssName: 'playbin',
     Properties: {
         uri: GObject.ParamSpec.string('uri', 'URI', '',
             GObject.ParamFlags.READWRITE,
-            ''
+            null
```

Comment:
JS doesn't have the concept of NULL strings, I'm not sure what will happen if you do this, but I think it might get marshalled as the empty string anyway, when setting the default value of the property. But I am pretty sure it's fine just to set `this._uri = null` in the `_init()` method as you do below, and leave this as-is.